uint16 command
float64 param1
float64 param2
float64 param3
float64 param4
float64 param5
float64 param6
float64 param7
uint8 target_system
uint8 target_component
uint8 confirmation
